prediction-polls/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,9 +24,9 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:usesCleartextTraffic="true"
         android:supportsRtl="true"
         android:theme="@style/Theme.PredictionPolls"
+        android:usesCleartextTraffic="true"
```

Comment:
We need to think about this flag before the final milestone. Let's talk with BE team tomorrow.